src/main/kotlin/no/nav/familie/tilbake/kravgrunnlag/batch/HåndterGamleKravgrunnlagBatch.kt

```diff
@@ -152,13 +152,13 @@ class HåndterGamleKravgrunnlagBatch(
                 Status.MANUELL_OPPFØLGING,
             )
 
-        val ALDERSGRENSE_I_UKER =
+        val ALDERSGRENSE_I_DAGER =
```

Comment:
Liker at vi endrer til dager - lurer litt på å skru ned for EF sin del til kortere tid, men kan være fint med "dager" for granuleringen sin del.